src/test/java/io/supertokens/test/oauth/OAuthStorageTest.java

```diff
@@ -316,12 +284,12 @@ public void testConstraints() throws Exception {
         } catch (TenantOrAppNotFoundException e) {
             // expected
         }
```

Comment:
why was this commented out? is this now expected to succeed? if so, reverse the test instead.

Reply:
fixed